src/dom.js

```diff
@@ -156,6 +184,15 @@ export function childElement(parent, callback) {
  * @return {?Element}
  */
 export function childElementByAttr(parent, attr, opt_value) {
+  if (scopeSelectorSupported == null) {
+    scopeSelectorSupported = isScopeSelectorSupported();
+  }
+  if (scopeSelectorSupported) {
+    if (opt_value !== undefined) {
+      return parent.querySelector(':scope [' + attr + '="' + opt_value + '"]');
```

Comment:
Important! Per API, this should be `:scope > [...]`, right? I.e. direct child. Or am I missing something about `:scope`?